checkout-ui-custom/src/controller/DeliverController.js

```diff
@@ -94,7 +94,7 @@ const DeliverController = (() => {
     if (window.location.hash === STEPS.SHIPPING) {
       setupDeliver();
       setCartClasses();
-
+      $('.bash--delivery-container').css('display', 'flex');
       $('.bash--delivery-container.hide').removeClass('hide');
```

Comment:
for some reason this class is added but the css is not applied, so i'm doing it with jquery. 

can change/remove this code once reviewed. @simmbiote